README.md

```diff
@@ -16,4 +16,5 @@ Movie search site that uses [OMDb API](http://omdbapi.com/) to get movies from I
 npm install
 npm start
 open http://localhost:3000
+go to omdb, get an apikey and paste it to the apiKey.js in the project src folder.
```

Comment:
I think this step should be before `npm start` since webpack crashes when I did these steps in this order :P